package.json

```diff
@@ -1,5 +1,6 @@
 {
   "name": "root",
+  "version": "0.60.0-SNAPSHOT",
```

Comment:
qq: Did it require this addition?

Reply:
Yes, I updated the description, but in short, the `ConfigService` is loaded before the acceptance tests framework, and without a version number the acceptance tests won't run.  

I really think the application version number needs to be required in order to allow the `web_clientVersion` method to work.  Making it required means it needs to be available in the `ConfigService` before the acceptance test framework is loaded.